### ISSUE TITLE
Strucutre

### ISSUE THREAD

**Issue by Mekki78** [PrestaShop/hipay#8]
Hello.

You should / must have a correct structure of your module, [img, gif, screenshot, etc] should be placed in /views/img and translation shoud be placed in /translations. 
Also you need to escape your variable in your .tpl files.

https://validator.prestashop.com

**Comment by jprotin** [comment]
Yes, it's under development. HiPay takes control of PrestaShop modules to ensure best quality.

The new files architecture and optimization code source will be operationnal mid-February before the Prestashop's validation. Go to => https://github.com/hipay/hipay-wallet-sdk-prestashop

Thank you
